.github/workflows/pull-request-title.yml

```diff
@@ -0,0 +1,32 @@
+name: Pull Request Title
+
+on:
+  pull_request:
+    types: [opened, edited]
```

Comment:
The title will be checked when the PR is opened or edited. However, a PR will also be seen as "edited" if its **body** is updated... and we can't fine tune this behavior.

But, given we cancel in-progress checks, a contributor won't be able to spam too much.